### PR TITLE
Add ssh keys setup to finish templates

### PR DESCRIPTION
**Pull request by ares**
Seems like these were forgotten when it was added.

### CODE REVIEW

**Comment by ares** [comment]
probably should be cherry-picked to 1.11 as well (ssh keys were introduced there)

File: preseed/finish.erb [hunk 1]
```diff
@@ -39,6 +39,8 @@ fi
 /usr/bin/puppet agent --config /etc/puppet/puppet.conf --onetime --tags no_such_tag <%= @host.puppetmaster.blank? ? '' : "--server #{@host.puppetmaster}" %> --no-daemonize
 <% end -%>
 
+<%= snippet('remote_execution_ssh_keys') %>
```
Comment (mmoll) [inline preseed/finish.erb]: it's already in line 59?

File: kickstart/finish.erb [hunk 2]
```diff
@@ -63,6 +63,8 @@ cat > /etc/puppet/puppet.conf << EOF
 <%= snippet 'puppet.conf' %>
 EOF
 
+<%= snippet('remote_execution_ssh_keys') %>
```
Comment (mmoll) [inline kickstart/finish.erb:66]: that's looking ok.

File: preseed/finish.erb (hunk 1 above)
Comment (ares) [inline preseed/finish.erb]: repetition is mother of all knowledge :see_no_evil:

:dart: thanks good catch

**Comment by mmoll** [comment]
merged and cherry-picked as ad01c08 for 1.11, díky @ares

**Pull request merged**